Sources/OpenAPI/OpenApiProvider.swift

```diff
@@ -0,0 +1,184 @@
+import Vapor
+
+public final class OpenApiProvider: Provider {
```

Comment:
```suggestion
public final class OpenAPIProvider: Provider {
```
According to Swift API design guidelines, acronyms that are commonly uppercased in English should be uppercased in code.

Reply:
Yeah, I switched to that style halfway through :P